libs/core/langchain_core/tools/base.py

```diff
@@ -460,7 +469,7 @@ def tool_call_schema(self) -> type[BaseModel]:
 
     def get_input_schema(
         self, config: Optional[RunnableConfig] = None
-    ) -> type[BaseModel]:
+    ) -> type[ArgsSchema]:
```

Comment:
this is not going to work because Runnables require pydantic output here. looking into a solution